src/sentry/api/serializers/models/project.py

```diff
@@ -364,13 +383,25 @@ def measure_span(op_tag):
             else:
                 recipient_actor = RpcActor.from_object(user)
             for project, serialized in result.items():
-                value = get_most_specific_notification_setting_value(
-                    notification_settings_by_scope,
-                    recipient=recipient_actor,
-                    parent_id=project.id,
-                    type=NotificationSettingTypes.ISSUE_ALERTS,
-                )
-                is_subscribed = value == NotificationSettingOptionValues.ALWAYS
+                # TODO(snigdha): why is this not included in the serializer
+                is_subscribed = False
+                if use_notifications_v2:
+                    provider_settings = notification_setting_options[project.id][
```

Comment:
@snigdhas could we just let the controller determine these? I think the goal of the controller is to handle as much of the logic as possible.